lib/elixir/lib/file.ex

```diff
@@ -712,8 +712,8 @@ defmodule File do
 
   Returns `:ok` in case of success, `{:error, reason}` otherwise.
 
-  Note: The command `mv` in Unix systems behaves differently depending
-  if `source` is a file and the `destination` is an existing directory.
+  Note: The command `mv` in Unix systems behaves differently depending on
+  whether `source` is a file and the `destination` is an existing directory.
   We have chosen to explicitly disallow this behaviour.
```

Comment:
We can probably get rid of this note altogether given the discussion on the forum. Up to you if you prefer to do it later though.

Reply:
I will make it part of a new PR